### PR TITLE
 Local

### LINKED ISSUE #31: Merge Sort

### PR DESCRIPTION
**Pull request by PHAGUN-JAIN**
mergesort

### CODE REVIEW

File: Mergesort/MergeSortPJ.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        i++;
+        k++;
+      }
+      else{
+      arr[k]=R[j];
+      j++;
+      k++;
+      }
+  }
+}
```
Comment (PHAGUN-JAIN) [inline Mergesort/MergeSortPJ.cpp:30]: merge two arrays at same time sorting!

Comment (AkshatGoyal1018) [reply Mergesort/MergeSortPJ.cpp:30]: if size1 = 2 and size2 = 3, then while function would terminate earlier but the last elements wouldn't copy in the array.
while(i<size1)
{
   arr[k]=L[i];
    k++;
     i++;
}
and same for size2

**Review by AkshatGoyal1018 — CHANGES_REQUESTED** [review]
Left the comment please see to it;

File: Mergesort/MergeSortPJ.cpp (hunk 1 above)
Comment (PHAGUN-JAIN) [reply Mergesort/MergeSortPJ.cpp]: taking your suggestion i understood the functionality. 
thanks 👍🏽

**Review by AkshatGoyal1018 — APPROVED** [review]
Nicely Done!

**Pull request merged**